Core/Main/itkTransformixFilter.hxx

```diff
@@ -237,6 +237,18 @@ TransformixFilter<TImage>::GenerateData()
     }
   }
 
+  if (m_ExternalTransform)
+  {
+    // External transforms should use "ResampleInterpolator" and the output image domain specification
+    // (Size/Spacing/Origin/Index/Direction) of the last transform parameter map.
+    auto transformParameterMap = std::move(transformParameterMapVector.back());
+    SetParameterValueAndWarnOnOverride(transformParameterMap, "Transform", "ExternalTransform");
+    SetParameterValueAndWarnOnOverride(
+      transformParameterMap, "TransformAddress", elx::Conversion::ObjectPtrToString(m_ExternalTransform));
+    transformParameterMapVector.clear();
+    transformParameterMapVector.push_back(std::move(transformParameterMap));
```

Comment:
@mstaring  Update: if an external transform is specified, only the _last_ map of the vector of transform parameter maps will now be taken into account. If the specified transform parameter object has `N` maps, the first `N - 1` maps will be ignored. OK?